packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-authjs/main/react/nutrition.md

```diff
@@ -1,4 +1,4 @@
-> **Note:** This document is only for Okta Identity Engine. If you’re using Okta Classic Engine, see [Sign in to SPA with Auth JS](/docs/guides/archive-sign-in-to-spa-authjs/react/main/). See [Identify your Okta solution](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-version) to determine your Okta version.
+> **Note:** This document is only for Okta Identity Engine. If you’re using Okta Classic Engine, see [Sign in to SPA with Auth JS](/docs/guides/archive-sign-in-to-spa-authjs/react/main/). See [Identify your Okta solution](https://help.okta.com/oie/en-us/Content/Topics/identity-engine/oie-verify-version.htm) to determine your Okta version.
```

Comment:
This alias is working, I don't think we need to hardcode the URL.

Reply:
See comment above